web/themes/custom/recipe_theme/src/sass/block/_block.scss

```diff
@@ -0,0 +1,12 @@
+.layout__region {
+  h2 {
+    display: flex;
+    justify-content: center;
+    margin: 2rem auto 1rem auto;
```

Comment:
this can be simplified to `margin: 2rem auto 1rem;
https://www.w3schools.com/css/css_margin.asp